.gitignore

```diff
@@ -0,0 +1,24 @@
+# Logs
```

Comment:
You can add .envs and generated files to this, perhaps refer to the old gitignore to check anything else that might be missing

Reply:
Added generated files, .env and logs to .gitignore